### PR TITLE
Updated pom to support pipeline operations

### PR DESCRIPTION
**Pull request by Loghijiaha**
## Types of changes

- [x] Infrastructure change (non-breaking change which updates dependencies or improves infrastructure)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
     <version>0.1-alpha-2-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.164.3</jenkins.version>
+        <jenkins.version>2.184</jenkins.version>
```
Comment (kinow) [inline pom.xml:39]: Would it be possible to match the LTS in the BOM 👇

Comment (Loghijiaha) [reply pom.xml:39]: there is no matching version for 2.184 LTS in bom. I changed to 2.190.x.

Comment (kinow) [reply pom.xml:39]: If our bill of materials if for Jenkins 2.190.x, we should probably support Jenkins 2.190.x LTS.

See git-plugin, for example. https://github.com/jenkinsci/git-plugin/blob/f06583ba2f83d7d15054ade0bfa68ab0009f611e/pom.xml#L32

They announce that the plugin should work with Jenkins at least at the version 2.204.1 (could work with earlier versions). And if you look at their bill of materials, they are using `bom-2.204.x`.

The supported Jenkins version normally matches the bill of materials, unless there's a good reason for doing other wise.

**Comment by Loghijiaha** [comment]
This is the CI error https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fmachine-learning-plugin/detail/PR-34/5/tests. I think this will be resolved after merging this PR because it shows that some plugin's version should be updated.

**Comment by kinow** [comment]
Not in front of my computer. Does mvn clean install works here Loghi? I think I tried today and it failed.

**Comment by Loghijiaha** [comment]
Last commit was successfull for me.

**Pull request merged**